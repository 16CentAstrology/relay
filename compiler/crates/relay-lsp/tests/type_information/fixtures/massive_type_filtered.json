{
  "type_name": "MassiveType",
  "string_filter": "10"
}
