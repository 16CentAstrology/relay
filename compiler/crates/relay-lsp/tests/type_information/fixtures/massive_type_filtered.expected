==================================== INPUT ====================================
{
  "type_name": "MassiveType",
  "string_filter": "10"
}
==================================== OUTPUT ===================================
type MassiveType {
  field10: String
  field100: String
  field101: String
  field102: String
  field103: String
  field104: String
  field105: String
  field106: String
  field107: String
  field108: String
  field109: String
  field110: String
  field210: String
  field310: String
  field410: String
  field510: String
  field610: String
  field710: String
  field810: String
  field910: String
  # 980 fields not matching filter
}
